web/components/dropdowns/project.tsx

```diff
@@ -23,6 +24,7 @@ type Props = TDropdownProps & {
   dropdownArrowClassName?: string;
   onChange: (val: string) => void;
   onClose?: () => void;
+  renderCondition?: (project: IProject) => boolean;
```

Comment:
name can be better, say getShouldRender or getShouldNotRender which describes the variable